_2021-finalproject/irvingskr/environment variable/check-environment.sh

```diff
@@ -0,0 +1,61 @@
+#判断环境变量是否存在
```

Comment:
Seems you lost something at the first line

Reply:
And your screenshot shold contain the entire process of run script and its output

Reply:
what is your first sentence's mean? Is it means that I should add the environment varieble setting sentence?

Reply:
No. I mean how do you run your script

Reply:
Oh,I know I lost the shebang , it's because I prefer to run the script with 'bash + file name'